.travis.yml

```
@@ -12,13 +12,14 @@ before_install:
   - docker-compose -f docker-compose-test.yml pull
   - docker-compose -f docker-compose-test.yml build api
   - docker-compose -f docker-compose-test.yml build data
-  - cd app && npm install && cd ..
+  - docker-compose -f docker-compose.yml build app
```

```suggestion
   - docker-compose -f docker-compose-test.yml build app
```

Currently, `docker-compose-test.yml` doesn't contain an `app` service, but one should be added. I think it should be the same as the `app` service in `docker-compose.yml`